tgg/research/gnomad_tgps.py

```diff
@@ -0,0 +1,239 @@
+"""This script checks the number of common (AF > 1%) variants per gnomAD sample. The
+gnomAD samples used are not present in the pangenome reference samples.
+
+Eimear Kenny (one of the co-chairs of the Human Pangenome Reference Consortium; HPRC)
+requested that we compute the number of common variants in v3 samples that aren't
+present in the pangenome reference samples. They want to use these counts (grouped by
+genetic ancestry group) to assess the new pangenome reference.
+
+Additional context from Eimear: Karen Miga (cc’d) and I are co-chairing the
+population sampling and representation (PSR) working group in the NHGRI-funded [Human
+Pangenome Reference Consortium]
+
+The goal of the consortium is to update the current human genome reference by
+producing a pangenome of high quality long read sequencing of 350 diverse humans
+which will become the new pangenome reference. The PSR group is responsible to
+selecting samples to be included in the pangenome, which should comprehensively cover
+most common variants, defined as variants >1%, in human populations globally. We are
+halfway through (phase 1) of the project, and most of the sampling has been selected
+from the 1 000 Genome Cell lines (N=174 samples). We have developed a simple variant
+counting metric that helps us evaluate anticipated performance of the pangenome using
+short read data in phase one samples and additional out-of-sample genomes from the
+1KG project. Here is a link to a brief slide deck that describes this: [
+https://www.dropbox.com/s/fo5nrxerxwukw7x/20220908.HPRC-PopulationSampling.Gnomad.pdf
+?dl=0]
+
+We would love to additionally be able to assess the performance in gnomAD – not only
+is this an independently ascertained dataset, but it would also give us a look ahead
+to how aligned the pangenome will be to the top resource used by clinical testing
+labs. Ideally, we would like to run the analysis on individual-level data in gnomAD,
+and I’m reaching out to see if you would be willing to collaborate with us on this
+analysis. The script is a very simple variant count algorithm, so hopefully very
+straightforward to run. We are currently writing up this work, and of course,
+anybody who participates in this analysis would be included as a co-author."""
+
+import hail as hl
+from gnomad_qc.v3.resources.basics import get_gnomad_v3_vds
+from gnomad_qc.v3.resources.release import release_sites
+from gnomad_qc.v3.resources.meta import meta
+import hailtop.fs as hfs
+import argparse
+import logging
+
+NUM_OF_GNOMAD_SAMPLES = 76156
+OUTPUT_FILENAME = "pangemoe_assessment_stats.txt"
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("Pangenome_assessment")
+logger.setLevel(logging.INFO)
+
+
+def get_pangenome_ids(
+        id_path: str,
```

Comment:
```suggestion
    id_path: str,
```
this seems to be indented a little too much? (not sure if it's just my screen)